test/no-content.html

```diff
@@ -91,7 +91,7 @@
                     changedEl = fixture('juicy-html-with-href').querySelector('juicy-html');
                     changedEl.setAttribute('href', './mock/200');
                     // wait for (faked) XHR
-                    setTimeout(done, 300);
+                    setTimeout(done, 900);
```

Comment:
Do we need this also in Sauce?

Reply:
Yes

Reply:
Then I'd rather test on Windows without this delay to make travis green to unlock further development. And keep linux support as an issue, 900ms is not the performance we aim for.

Reply:
Now it is working on Linux && Chrome && without a delay! Driving me crazy. I pushed a commit I hope it will act the same so you could see.

Reply:
The above comment is incorrect. I removed the delay from a wrong place.